### PR TITLE
Add Sharesite as an official plugin

### PR DESCRIPTION
**Pull request by Thynix**
Thanks to ArneBab for reviewing it and readying it for inclusion!

### CODE REVIEW

File: src/freenet/l10n/freenet.l10n.en.properties [hunk 1]
```diff
@@ -1473,6 +1473,7 @@ PluginManager.pluginDesc.Freetalk=Experimental anonymous forum. Currently has se
 PluginManager.pluginDesc.WebOfTrust=Allows you to create an anonymous identity. These identities can trust each other, or not, and hence avoid spam and DoS attacks. Backend for chat and social applications over Freenet.
 PluginManager.pluginDesc.WebOfTrustTesting=Very new versions of the Web of Trust plugin, for testing purposes. Compiled by developer "xor". ATTENTION: These are not compiled on a high security computer. There is no key revocation mechanism for their update USK - if it gets hacked, you are in trouble. Updates will be installed at every restart of Freenet, even if auto-updating is disabled in configuration! Can only be fetched over Freenet, not from our central servers. Please unload WebOfTrust first before trying to load this plugin.
 PluginManager.pluginDesc.FlogHelper=Easy to use tool for creating a blog on Freenet. Uses WebOfTrust to automatically announce your new flog so it should show up on the freesite indexes.
+PluginManager.pluginDesc.Sharesite=Tool to publish and update single-page sites based on ShareWiki.
```
Comment (Thynix) [inline src/freenet/l10n/freenet.l10n.en.properties]: This phrasing seems awkward reading it again. Maybe

> Tool to publish and update single-page sites. Based on ShareWiki.

?

**Comment by nathanielnebel** [comment]
I will agree that the phrasing is a little awkward. How about this, "Enables you to publish and update single-page sites built with ShareWiki?"

File: src/freenet/l10n/freenet.l10n.en.properties (hunk 1 above)
Comment (ArneBab) [inline src/freenet/l10n/freenet.l10n.en.properties]: Maybe simply

> Publish and update single-page websites. Based on ShareWiki.

?

I just realized that our descriptions of plugins do not follow a consistent pattern. UPnP vs. FlogHelper vs. WebOfTrust — very different ways to describe a plugin. But that’s a fix for another day…

File: src/freenet/pluginmanager/OfficialPlugins.java [hunk 2]
```diff
@@ -123,6 +123,10 @@ public OfficialPlugins() {
 					.minimumVersion(35)
 					.usesXml()
 					.loadedFrom("CHK@3Ht6otYHogKxa1Z8SC0hz46kV9q1qaSBsjbJxuwPosU,txky3jF33oWU9n6YiGMphiHA1V9q3i4ZN0VPtgElSaI,AAMC--8/FlogHelper-v35.jar");
+			addPlugin("Sharesite")
+					.inGroup("communication")
+					.minimumVersion(2)
+					.loadedFrom("CHK@F2HY72tkYOCjn6JxipG3jumisfczBlDgn7HhQ~tz1zg,3ab-60XR~Zvcidvs0Qjb7rygTTXVrfx9A~3kZBvYT4k,AAMC--8/Sharesite-v0.2.7.jar");
```
Comment (ArneBab) [inline src/freenet/pluginmanager/OfficialPlugins.java]: fetching the CHK works

**Comment by Thynix** [comment]
@nathanielnebel That's part of the reason it's awkward. There are two separate things:

1. The plugin allows publishing and updating single-page sites.
2. This plugin is based on the plugin ShareWiki.

**Pull request merged**